src/courses/forms.py

```diff
@@ -1,14 +1,46 @@
 from django import forms
 
 from crispy_forms.helper import FormHelper
-from crispy_forms.layout import Layout
-from crispy_forms.bootstrap import Div
+from crispy_forms.layout import HTML, Div, Layout
+from crispy_forms.bootstrap import Accordion, AccordionGroup
 from bootstrap_datepicker_plus.widgets import DateTimePickerInput, TimePickerInput
 
-from .models import Block, Course, CourseStudent, Semester, ExcludedDate
+from .models import Block, Course, CourseStudent, MarkRange, Semester, ExcludedDate
 from siteconfig.models import SiteConfig
 
 
+class MarkRangeForm(forms.ModelForm):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields['courses'].help_text = "Which courses this field is relevant to; If left blank it will apply to all courses. Hold down \
```

Comment:
Rather than duplicate the existing help_text, get the existing help text and append it with the extra widget multi-select info? That will future proof this in case the help text changes on the model.

Something like https://stackoverflow.com/questions/50496895/how-to-get-help-text-on-a-custom-django-modelchoicefield

Reply:
Actually, can you change the help text on the model field to:
"Which courses this Mark Range will be used for; if left blank it will apply to all courses."

Reply:
model field text changed and dynamic help text implemented!